src/core/load-fragment.c

```diff
@@ -2390,6 +2390,375 @@ int config_parse_documentation(const char *unit,
 }
 
 #ifdef HAVE_SECCOMP
+static int syscall_filter_parse_one(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                ExecContext *c,
+                bool invert,
+                const char *t,
+                bool warn) {
+        int r;
+
+        if (*t == '@') {
+                const char * const *s;
+                static const char * const syscall_sets[] = {
+                        /* Default list */
+                        "@Default",
+                        "execve\0"
+                        "exit\0"
+                        "exit_group\0"
+                        "rt_sigreturn\0"
+                        "sigreturn\0",
+
+                        /* Need superuser capabilities */
+                        "@SysAdmin",
```

Comment:
I agree for the permitting side. But this group is still useful when denying all the system admin calls for unprivileged units.